src/Digdir.Domain.Dialogporten.Application/ApplicationExtensions.cs

```diff
@@ -48,11 +50,16 @@ public static IServiceCollection AddApplication(this IServiceCollection services
         }
 
         var localDeveloperSettings = configuration.GetLocalDevelopmentSettings();
-        services.Decorate<IUserService, LocalDevelopmentUserServiceDecorator>(
+        services.Decorate<IUserResourceRegistry, LocalDevelopmentUserResourceRegistryDecorator>(
             predicate:
             localDeveloperSettings.UseLocalDevelopmentUser ||
             localDeveloperSettings.UseLocalDevelopmentResourceRegister);
 
+        services.Decorate<IUserNameRegistry, LocalDevelopmentUserNameRegistryDecorator>(
+            predicate:
+            localDeveloperSettings.UseLocalDevelopmentUser ||
+            localDeveloperSettings.UseLocalDevelopmentNameRegister);
+
```

Comment:
Burde IUserOrganizationRegistry ha sin egen lokale?

Reply:
Den har ingenting som trenger dekorasjon, local dev user har et claim for org.nr

Reply:
done